cirq_superstaq/ops/qubit_gates_test.py

```diff
@@ -191,21 +195,48 @@ def test_acecr_circuit_diagram_info() -> None:
         1: ───AceCR+-(Z side)───"""
     )
 
-    circuit = cirq.Circuit(css.AceCR("+-", np.pi)(*qubits))
+    circuit = cirq.Circuit(css.AceCR("+-", sandwich_rx_rads=np.pi)(*qubits))
     assert str(circuit) == textwrap.dedent(
         """\
         0: ───AceCR+-(Z side)──────────
               │
         1: ───AceCR+-(X side)|Rx(π)|───"""
     )
 
+    circuit = cirq.Circuit(css.AceCR(theta=np.pi)(*qubits))
+    assert str(circuit) == textwrap.dedent(
+        """\
+        0: ───AceCR(π)(Z side)───
+              │
+        1: ───AceCR(π)(X side)───"""
+    )
+
+    circuit = cirq.Circuit(css.AceCR(theta=np.pi / 3)(*qubits))
+    assert str(circuit) == textwrap.dedent(
+        """\
+        0: ───AceCR(0.333π)(Z side)───
+              │
+        1: ───AceCR(0.333π)(X side)───"""
+    )
+
+    circuit = cirq.Circuit(css.AceCR(theta=np.pi, sandwich_rx_rads=np.pi)(*qubits))
+    assert str(circuit) == textwrap.dedent(
+        """\
+        0: ───AceCR(π)(Z side)──────────
+              │
+        1: ───AceCR(π)(X side)|Rx(π)|───"""
+    )
+
 
 def test_acecr_qasm() -> None:
     qubits = cirq.LineQubit.range(2)
     circuit = cirq.Circuit(
         css.AceCR("+-").on(*qubits),
         css.AceCR("-+").on(*reversed(qubits)),
-        css.AceCR("-+", np.pi / 2).on(*qubits),
+        css.AceCR("+-", sandwich_rx_rads=np.pi / 2).on(*qubits),
+        css.AceCR("-+", sandwich_rx_rads=np.pi / 2).on(*qubits),
+        css.AceCR(theta=np.pi / 3).on(*qubits),
+        css.AceCR(theta=np.pi / 3, sandwich_rx_rads=np.pi / 2).on(*qubits),
```

Comment:
nit: can we use a different denominator here, e.g.
```suggestion
        css.AceCR(theta=np.pi / 5).on(*qubits),
        css.AceCR(theta=np.pi / 5, sandwich_rx_rads=np.pi / 2).on(*qubits),
```
so that the strings below don't depend on the default precision of the printer?